scipy/stats/_stats_py.py

```diff
@@ -7501,48 +7501,84 @@ def chisquare(f_obs, f_exp=None, ddof=0, axis=0):
            in the case of a correlated system of variables is such that it can be reasonably
            supposed to have arisen from random sampling", Philosophical Magazine. Series 5. 50
            (1900), pp. 157-175.
+    .. [4] Mannan, R. William and E. Charles. Meslow. “Bird populations and
+           vegetation characteristics in managed and old-growth forests,
+           northeastern Oregon.” Journal of Wildlife Management
+           48, 1219-1238, :doi:`10.2307/3801783`, 1984.
 
     Examples
     --------
-    When just `f_obs` is given, it is assumed that the expected frequencies
-    are uniform and given by the mean of the observed frequencies.
+    In [4]_ bird foraging behavior was investigated in a forest of Oregon.
+    The foraging refers to the range of activities and behaviours exhibited by
+    birds in their quest for food.
+    In an old-growth forest, 44% of the canopy volume was Douglas fir,
+    24% was ponderosa pine, 29% was grand fir, and 3% was western larch.
+    They observed the behavior of several species of birds, one of which
+    was the red-breasted nuthatches. They made 189 observations of foraging
+    of this specie with the respective frequencies of 23%, 27%, 29% and 21%.
+
+    Using a chi-square test, we can test the null hypothesis that the
+    proportions of foraging events are equal to the proportions of canopy
+    volume. Let's consider a significance level of 5%.
+
+    Using the above proportions of canopy volume and observed events, we can
+    infer expected frequencies.
 
     >>> import numpy as np
+    >>> f_exp = np.array([44, 24, 29, 3]) / 100 * 189
+    >>> f_obs = np.array([23, 27, 29, 21]) / 100 * 189
+
+    We can now confront the observed frequencies with the expected frequencies
```

Comment:
I think we made this change in one of the other doc PRs.
```suggestion
    We can now compare the observed frequencies with the expected frequencies.
```
https://www.merriam-webster.com/dictionary/confront